java/pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>org.apache.arrow</groupId>
   <artifactId>arrow-java-root</artifactId>
-  <version>siren-4.0.0-1</version>
+  <version>siren-4.0.0-1-SNAPSHOT</version>
```

Comment:
Shouldn't it be `siren-4.0.0-2-SNAPSHOT` now that `siren-4.0.0-1` is released ?

Reply:
Yes it should be `siren-4.0.0.2-SNAPSHOT`, done https://github.com/sirensolutions/arrow/pull/18/commits/b62a621a12fd46eb5ad4071ccc952554cf8a77b8 thanks